packaging/suse/make_spec.sh

```diff
@@ -54,42 +54,52 @@ Name:           $NAME
 Version:        $VERSION
 Release:        0
 License:        Apache-2.0
-Summary:        Automatically load all docker images that are packaged in RPM
-Url:            https://%{import_path}
+Summary:        Load container images from RPMs into different container engines
+Url:            https://github.com/kubic-project/container-feeder
 Group:          System/Management
 Source:         ${SAFE_BRANCH}.tar.gz
 Source1:        sysconfig.%{name}
 Source2:        %{name}.service
-BuildRequires:  golang-packaging systemd
+Source3:        %{name}-rpmlintrc
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
+BuildRequires:  device-mapper-devel
+BuildRequires:  fdupes
+BuildRequires:  glib2-devel-static
+BuildRequires:  glibc-devel-static
+BuildRequires:  go-go-md2man
+BuildRequires:  golang-packaging
+BuildRequires:  libapparmor-devel
+BuildRequires:  libassuan-devel
+BuildRequires:  libbtrfs-devel
+BuildRequires:  libgpgme-devel
+BuildRequires:  libseccomp-devel
+BuildRequires:  golang(API) >= 1.7
 Requires:       docker-kubic
+Requires:       libcontainers-common
+Requires:       libcontainers-image
+Requires:       libcontainers-storage
```

Comment:
All those are required for containers/storage and images etc.

Reply:
You dropped the systemd dependency